pkg/runtime/cache/namespace.go

```diff
@@ -177,6 +201,17 @@ func (c *NamespaceCache) setNamespaceInfoFromK8sObject(ns *corev1.Namespace) {
 	if ok {
 		nsInfo.endpointURL = EndpointURL
 	}
+
+	nsInfo.deletionPolicies = map[string]string{}
+	nsDeletionPolicySuffix := "." + ackv1alpha1.AnnotationDeletionPolicy
```

Comment:
nit:  Very tiny optimization: we can declare this as a constant outside of this function to avoid making unnecessary allocations everytime the cache syncs